electron.js

```diff
@@ -1,12 +1,13 @@
-const electron = require('electron');
+const electron = require('electron')
 // Module to control application life.
-const app = electron.app;
+const app = electron.app
 // Module to create native browser window.
-const BrowserWindow = electron.BrowserWindow;
-
+const BrowserWindow = electron.BrowserWindow
+const REDUX_DEV_TOOLS = require('./config/config').REDUX_DEV_TOOLS
```

Comment:
ripristinato.
si gestirà tramite conflitto nel merge in cui si recupera la versione originale.